nbviewer/handlers.py

```diff
@@ -693,8 +693,12 @@ def get(self, user, gist_id, filename=''):
             entries = []
             ipynbs = []
             others = []
+            
+            iter_files = (
+                getattr(files, "itervalues", None) or getattr(files, "values")
+            )
 
-            for file in files.itervalues():
+            for file in iter_files():
```

Comment:
My Python 3-centrism says to use `files.values()` in places like this, and let Python 2 be slightly less efficient than Python 3. These aren't large enough containers for the optimization to ever be critical.